main.py

```diff
@@ -26,6 +26,12 @@
 def log(message):
     print(message)
 
+# Change color of LED only if P2 not in Use for AP
+def rgb_led(rgb):
+    if not(_config.get_value('general', 'general', 'button_ap_enabled') 
```

Comment:
not(_config.get_value('general', 'general', 'button_ap_enabled') 

die Abfrage brauchen wir ja eigentlich nicht, es reicht ja, wenn es nicht pin 2 ist, oder?